src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLoadContext.cs

```diff
@@ -50,6 +55,37 @@ internal class PackageLoadContext
 
         public Task<PackageCollection> GetInstalledPackagesAsync() =>_installedPackagesTask;
 
+        public async Task<List<string>> GetTargetFrameworksAsync()
```

Comment:
I think we can remove this and not do it in V1 because we are not doing PR in v1.

Reply:
Removed